iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -1626,6 +1630,12 @@ + (void)load {
     [OneSignalHelper registerAsUNNotificationCenterDelegate];
 }
 
++(BOOL) shouldDisableBasedOnProcessArguments {
+    if ([[[NSProcessInfo processInfo]arguments]containsObject: @"DISABLE_ONESIGNAL"]){
```

Comment:
The name `DISABLE_ONESIGNAL` implies it is completely disabling OneSignal. I thinking it needs to be a bit more verbose as it is only disabling swizzling.
@depoon What do you think of `DISABLE_ONESIGNAL_SWIZZLING`?

Reply:
Lets go with your choice of name.